peps/pep-0772.rst

```diff
@@ -184,201 +189,182 @@ making process that governs the technical projects under the PyPA umbrella.
+Packaging Council elections every year.
 
 .. _process:
 
 Processes
 =========
 
-Initial membership
-------------------
-
-Initial set of voting members will include anyone who has taken the time to
-formalise their participation in the packaging community. This includes:
-
-* PyPA members: Anyone with the triage bit or commit bit, or at least one
-  project in the PyPA organisation.
-* Packaging workgroup members: Anyone who is listed on the Packaging WG charter
-  who is willing to participate is welcome.
-* Interested core team members: Any Python core team member who is willing to
-  participate is welcome.
-* Wider community members: An initial set of for-profit companies, nonprofit
-  organizations, academic or educational institutions and smaller unaffiliated
-  projects would be invited to nominate three individuals to represent them.
-  See :ref:`the initial set of community voting members <appendix_C>`.
-
-Adding a new member
--------------------
-
-New voting members are added by a simple majority vote by the current
-membership. Quorum for adding new members is 50%.
-
-A vote to add a new member is triggered when a voting member calls for one
-publicly on an appropriate communication channel, and another voting member
-seconds the call within two weeks.
-
-The vote lasts for one week. Each member can vote for or against.
-
 Removal of a member
 -------------------
 
-In order to maintain a reasonable expectation of quorum, failure to participate
-in Packaging Council elections for two consecutive council elections
-automatically removes a person from the list of voting members, until they
-re-submit their intention to resume their participation to the Packaging
-Council in writing.
-
-In exceptional circumstances, it may be necessary to remove someone from the
-voting members against their will (for example: egregious and ongoing code of
-conduct violations). A voting member may be removed by a two-thirds majority
-vote by the Packaging Council (in practice: 4:1 for a council with five
-members).
-
-If the relevant voting member is also on the Packaging Council, then they can
-participate in the vote. They are removed from the Packaging Council if the
-vote removes them as a voting member. The vacancy is filled as per the process
-for filling vacancies in the Packaging Council.
+In order to maintain a reasonable expectation of quorum, failure to participate in Packaging Council elections
+for two consecutive Council elections automatically removes a person from the list of Packaging Council
+Electors, until they re-submit their intention to resume their participation to the Packaging Council in
+writing.
 
 Vote of no confidence
 ---------------------
 
-In exceptional circumstances, the voting members may remove a sitting council
-member, or the entire council, via a vote of no confidence.
+In exception circumstances, a vote of no confidence may be called to remove a sitting Packaging
```

Comment:
```suggestion
In exceptional circumstances, a vote of no confidence may be called to remove a sitting Packaging
```

Reply:
Thanks!  Fixed locally to avoid merge conflicts.